pkg/presentation/mappers/mappers.go

```diff
@@ -102,6 +108,9 @@ func EmployeeToViewModel(entity *employee.Employee) *viewmodels.Employee {
 }
 
 func UploadToViewModel(entity *upload.Upload) *viewmodels.Upload {
+	if entity == nil {
```

Comment:
Here as well, I think it's better to keep these functions seg faulting when handling nil pointers